src/migration/ember-addon/steps/update-addon-tsconfig-json.js

```diff
@@ -28,10 +29,7 @@ export function updateAddonTsconfigJson(options) {
 
   const oldPath = join(projectRoot, locations.addon, 'tsconfig.json');
   const oldFile = readFileSync(oldPath, 'utf8');
-  const tsconfigJson = JSON.parse(
-    // Remove comments
-    oldFile.replace(new RegExp('//.*', 'mg'), '')
-  );
+  const tsconfigJson = JSON.parse(sanitizeJson(oldFile));
```

Comment:
I'll make a separate pull request for re-organizing (and perhaps simplifying) the utility functions.